### PR TITLE
+interval 1.3

### PR DESCRIPTION
**Pull request by Chris00**

### CODE REVIEW

**Comment by dbuenzli** [comment]
This looks like a binary release. opam is a source based package manager. Could please explain what is going on ?

**Comment by Chris00** [comment]
@dbuenzli It is not a binary release—why do you think it is?

**Comment by Chris00** [comment]
Good to merge!

**Comment by dbuenzli** [comment]
> @dbuenzli It is not a binary release—why do you think it is?

Sorry I thought I saw an `ocamlfind install` for objects without any `build:` invocation. It seems I dreamt.

File: packages/interval/interval.1.3/files/META [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name = "interval"
```
Comment (samoht) [inline packages/interval/interval.1.3/files/META]: would probably be a good idea to push that file upstream if possible

Comment (Chris00) [inline packages/interval/interval.1.3/files/META]: I agree.  I am discussing with the authors but in order not to delay the initial release, I put it in `files/`.

**Pull request merged**